### DISCUSSION TITLE
How do I display history notifications?

### DISCUSSION THREAD

**Discussion by iamxiaojianzheng** [folke/snacks.nvim#56] [Q&A]
same title

**Comment by folke** [comment]
It's in the [docs](https://github.com/folke/snacks.nvim/blob/main/docs/notifier.md)

**Reply by iamxiaojianzheng** [reply]
![image](https://github.com/user-attachments/assets/7898eee3-0ce1-4d3a-8895-5aaf9a814ad3)


```lua
Snacks.notifier.show_history()
```

![image](https://github.com/user-attachments/assets/6377c22c-bb3e-4e70-82b1-0b79e0b5f6cb)

**Reply by iamxiaojianzheng** [reply]
View is empty

**Comment by zjp-CN** [comment]
The command is `:lua Snacks.notifier.show_history()` , which works for me.
